ACE/ace/OS_NS_signal.inl

```diff
@@ -15,6 +15,14 @@ kill (pid_t pid, int signum)
   ACE_UNUSED_ARG (pid);
   ACE_UNUSED_ARG (signum);
   ACE_NOTSUP_RETURN (-1);
+#elif defined (ACE_VXWORKS)
+  /*
```

Comment:
That's fine - the only difference is the signature for kill() on VxWorks, and it should work regardless. This change can stay unless/until someone with a VxWorks build environment can test it.